data/Building/comment.js

```diff
@@ -33,6 +35,11 @@ async function validateAndCleanComment(buildingId, posterId, message) {
   if (!(await Building.exists({_id: buildingId}))) {
     throw new Error('Cannot post review to nonexistent building');
   }
+
+  // Filtering bad words with stars
+  const filter = new Filter();
+  filter.clean(message);
```

Comment:
```js
filter.clean(message)
``` 

cleans the message, but it doesn't save the cleaned message (pass-by-value). This needs to be changed to 

```js
message = filter.clean(message)`
```